whac-a-mole.py

```diff
@@ -198,6 +207,8 @@ def whac_a_mole():
             sleep(1)
             hole9.config(state='disabled', text='', bg='black')
     replaybtn.config(state='normal')
+    winsound.PlaySound('tada1.wav',winsound.SND_FILENAME)

```

Comment:
Please check for PEP-8 compliance in all parts (it's breaking in certain parts of code). 
You can use `pycodestyle` to check for compliance. 
(Install pycodestyle through pip)